templates/tasseo.conf.j2

```diff
@@ -7,6 +7,8 @@ respawn
 setuid tasseo
 chdir {{ tasseo_dir }}/tasseo-{{ tasseo_version }}
 
-env GRAPHITE_URL={{ tasseo_graphite_url }}
+script
+  export GRAPHITE_URL="{{ tasseo_graphite_url }}"
 
-exec bundle exec rackup -I lib -p {{ tasseo_port }} -s thin >> {{ tasseo_log }} 2>&1
+  exec bundle exec rackup -I lib -p 5000 -s thin >> /var/log/tasseo.log 2>&1
```

Comment:
Do `tasseo_port` and `tasseo_log` still need to be here? If not, they should probably be removed from `defaults.yml`